docs/data/sql_functions.yml

```diff
@@ -377,6 +377,12 @@ string:
   - sql: SUBSTR(string, integer1[, integer2])
     table: STRING.substr(INTEGER1[, INTEGER2])
     description: Returns a substring of string starting from position integer1 with length integer2 (to the end by default).
+  - sql: JSON_QUOTE(string)
+    table: STRING.JsonQuote()
+    description: Quotes a string as a JSON value by wrapping it with double quote characters, escaping interior quote and other characters, and returning the result as a utf8mb4 string. If the argument is NULL, the function returns NULL.
+  - sql: JSON_UNQUOTE(string)
+    table: STRING.JsonUnquote()
+    description: Unquotes JSON value and returns the result as a utf8mb4 string. If the argument is NULL, returns NULL. If the value starts and ends with double quotes but is not a valid JSON string literal, the value is passed through unmodified.
```

Comment:
IMHO That's looks like a questionable behaviour I would suggest to discuss it first in  jira.
It could be quite obvious to see whether it ws unquoted or or just passed through as unmodified however it could be very tricky for complex strings

Thoughts out loud: By the was we already have `IS JSON`, couldn't we reuse it

Reply:
Yes, I also had a similar question. From the definition, if the input is not a valid JSON string literal, then we pass the value as it is. For example, `"key"` is not a valid JSON literal but `["key"]` is. Maybe we need more investigation on this.